test/abstractarray.jl

```diff
@@ -201,6 +201,22 @@ using StaticArrays, Test, LinearAlgebra
     end
 end
 
+@testset "permutedims" begin
+    @test @inferred(permutedims(SVector(1,2,3))) === SMatrix{1,3}(1,2,3)
+    @test @inferred(permutedims(MVector(1,2,3))) == MMatrix{1,3}(1,2,3)
+    @test @inferred(permutedims(MVector(1,2,3))) isa MMatrix{1,3}
+    @test @inferred(permutedims(SizedVector{3}(rand(3)))) isa SizedMatrix{1,3}
```

Comment:
Could you add a test like this here: `@test permutedims(SizedVector{3}([1,2,3])) == [1 2 3]` ?